hnn_core/gui.py

```diff
@@ -0,0 +1,374 @@
+"""IPywidgets GUI."""
+
+# Authors: Mainak Jas <mjas@mgh.harvard.edu>
+
+import json
+import os.path as op
+from functools import partial, update_wrapper
+
+import numpy as np
+
+import hnn_core
+from hnn_core import simulate_dipole, read_params, Network
+
+from IPython.display import display
+
+from ipywidgets import (FloatLogSlider, Dropdown, Button, RadioButtons,
+                        fixed, interactive_output, interactive, interact,
+                        FloatText, IntText, FileUpload, HTML, Output,
+                        HBox, VBox, Tab, Accordion,
+                        Layout, AppLayout)
+
+
+def create_expanded_button(description, button_style, height):
+    style = {'button_color': '#8A2BE2'}
+    return Button(description=description, button_style=button_style,
+                  layout=Layout(height=height, width='auto'),
+                  style=style)
+
+
+def update_params(params, **updates):
+    params.update(dict(**updates))
+    return params
+
+
+def _get_sliders(params, param_keys):
+    """Get sliders"""
+    style = {'description_width': '150px'}
+    sliders = list()
+    for d in param_keys:
+        slider = FloatLogSlider(
+            value=params[d], min=-5, max=1, step=0.2,
```

Comment:
The dynamic range for neurotransmitters may differ between neurotransmitters, might be worth considering different mix/maxes for GABA vs. AMPA

Reply:
Note this in log scale! But yes totally open to suggestions to improve the interface here